support/android/apk/servoview/src/main/java/com/mozilla/servoview/Servo.java

```diff
@@ -27,7 +30,9 @@ public Servo(
             String args,
             String url,
             String logstr,
-            int width, int height, float density, boolean log) {
+            int width, int height,
+            float density, boolean log,
+            Context context) {
 
```

Comment:
Context and activity are the same objects. Can you only pass one and cast activity to context or context to activity?

Reply:
Ugh, of course. I'm blind.